src/cheri-pte-ext.adoc

```diff
@@ -121,14 +123,40 @@ and the virtual page's CRG bit does not equal senvcfg.CRG.
 NOTE: An implementation may avoid a dependency on the tag
 bit value of the capability read. This will introduce additional traps during revocation sweeps.
 
+[[pte_cw_crg_load_summary]]
+.Summary of Load CW and CRG behavior in the PTEs
+[%autowidth,float="center",align="left",cols="<,<,<",options="header"]
+|===
+|PTE.CW |PTE.CRG           |Load/AMO
+| 0     | X                | Clear loaded tag
+| 1     |!=<<senvcfg>>.CRE | Page fault, or page fault if tag is set
+| 1     |==<<senvcfg>>.CRE | Normal operation
+|===
+
+[[pte_cw_crg_store_summary]]
+.Summary of Store CW and CRG behavior in the PTEs
+[%autowidth,float="center",align="left",cols="<,<,<",options="header"]
+|===
+|PTE.CW |PTE.CRG           |Store/AMO
+| 0     | 0                | Page fault if stored tag is set
+| 0     | 1                | Page fault if stored tag is set, or hardware CW, CRG update
+| 1     | X                | Normal operation
+|===
+
+
 [#senvcfg_pte,reftext="senvcfg PTE bits"]
-=== Extending the Supervisor Environment Configuration Register (menvcfg)
+=== Extending the Machine and Supervisor Environment Configuration Registers (menvcfg/senvcfg)
 
-The *senvcfg* register operates as described in cite:[riscv-priv-spec].
+The *menvcfg* and *senvcfg* register operate as described in cite:[riscv-priv-spec].
 {cheri_pte_ext_name} adds a new Capability Read Generation (CRG) bit as shown
-in xref:senvcfgreg_pte[xrefstyle=short] when XLEN=64. The CRG bit in *senvcfg*
+in xref:menvcfgreg_pte[xrefstyle=short] and xref:senvcfgreg_pte[xrefstyle=short]
+when XLEN=64. The CRG bit in *senvcfg*
 is used as described in xref:section_extending_pte[xrefstyle=short].
```

Comment:
Does that mean that the CRG bit is mirrored in both `senvcfg` and `menvcfg`? The extension would not work well if there were multiple, differing  instances of the bit around.

Reply:
it's the same as all other bits in Xengcfg - it's the same bit